src/dataDisplay/comment/comment.jsx

```diff
@@ -140,99 +213,109 @@ class Comment extends React.Component {
                             }}
                         >
                             <span className="comment-name">{name}</span>
-                            <span className="comment-time">{this._renderTime()}</span>
+                            <span className="comment-time">{this.renderTime()}</span>
                         </div>
-                        { isEditable && (canEdit || canDelete) ?
-                            isEditMode ? (
-                                <div
+
+                        {isEditable && (canEdit || canDelete) && isEditMode && (
+                            <div
+                                style={{
+                                    flex: '1 0 auto',
+                                    margin: 0,
+                                    paddingTop: '19px',
+                                    textAlign: editActionMenuAlignment,
+                                }}
+                            >
+                                <A
+                                    className="cancel-link font-size-xsmall color-text"
+                                    onClick={this.onCancelEditClick}
+                                    onKeyDown={noop}
                                     style={{
-                                        flex: '1 0 auto',
-                                        margin: 0,
-                                        paddingTop: '19px',
-                                        textAlign: editActionMenuAlignment,
+                                        display: 'inline-block',
                                     }}
+                                    tabIndex={0}
                                 >
-                                    <a
-                                        className="cancel-link font-size-xsmall color-text"
-                                        onClick={this.onCancelEditClick}
-                                        style={{ display: 'inline-block' }}
-                                    >
-                                        Cancel
-                                    </a>
-                                    <a
-                                        className="save-link font-size-xsmall"
-                                        disabled={!canEdit}
-                                        onClick={this.onSaveClick}
-                                        style={{ display: 'inline-block', marginLeft: '22px' }}
-                                    >
-                                        Save
-                                    </a>
-                                </div>
-                            ) : (
-                                <div
+                                    Cancel
+                                </A>
+
+                                {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
```

Comment:
We can probably lose this now that we're using `<A>` rather than `<a>`

Reply:
Fixed.

Reply:
I'm thinking we can remove `{/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}` now.

Reply:
Gone now.  Resolving.